cdbcli/commands.py

```diff
@@ -190,7 +190,10 @@ def mkdir(environment, couch_server, variables):
     if database_name in couch_server:
         raise RuntimeError('Database {} already exists'.format(database_name))
 
-    couch_server.create(database_name)
+    try:
+        couch_server.create(database_name)
+    except couchdb.Unauthorized as e:
+        raise RuntimeError(e)
```

Comment:
Since we print out the message of `RuntimeError`s, I'd prefer if we send `str(e)` to `RuntimeError()` as opposed to sending the exception object.